src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -555,19 +577,22 @@ def run_single_testcase(self, testcase_path):
 
     return result
 
+  def set_environment_variable(self, key, val):
+    environment.set_value(key, val)
+
   def set_environment_variables(self):
     """Sets environment variables needed by afl."""
     # Tell afl_driver to duplicate stderr to STDERR_FILENAME.
     # Environment variable names and values that must be set before running afl.
-    environment.set_value(constants.FORKSRV_INIT_TMOUT_ENV_VAR,
-                          constants.FORKSERVER_TIMEOUT)
-    environment.set_value(constants.FAST_CAL_ENV_VAR, 1)
-    environment.set_value(constants.IGNORE_UNKNOWN_ENVS_ENV_VAR, 1)
-    environment.set_value(constants.SKIP_CRASHES_ENV_VAR, 1)
-    environment.set_value(constants.SKIP_CPUFREQ_ENV_VAR, 1)
-    environment.set_value(constants.BENCH_UNTIL_CRASH_ENV_VAR, 1)
-    environment.set_value(constants.STDERR_FILENAME_ENV_VAR,
-                          self.stderr_file_path)
+    self.set_environment_variable(constants.FORKSRV_INIT_TMOUT_ENV_VAR,
```

Comment:
Rather than do this, could we just iterate through all host env vars and collect the ones starting with `AFL_` to forward to the device?